src/dal_select2/static/autocomplete_light/select2.js

```diff
@@ -8,13 +8,24 @@
             var data_forward = {};
 
             for (var key in forward) {
+                var f = forward[key].split("->");
+                var src_name, dst_name;
+                if (f.length === 0) {
+                    continue;
+                } else if (f.length === 1) {
+                    src_name = f[0];
+                    dst_name = f[0];
+                } else {
+                    src_name = f.splice(0,1);
+                    dst_name = f.join("->");
+                }
```

Comment:
> and have it encapsulated in a function in autocomplete.js

I think the whole add_forwards() function should be outside `select2.js` since it has no select2-specific functionality.